CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # webkms-switch ChangeLog
 
+## 4.0.0 - 
+
+### Changed
+- **BREAKING**: Use latest version of `http-signature-zcap-verify`.
```

Comment:
Maybe subitems:
- Only support `Ed25519Signature2020` proofs.
- Node.js >= 12 is required
- https://github.com/digitalbazaar/http-signature-zcap-verify/blob/main/CHANGELOG.md

Reply:
Updated c3e1c23